shared/api/storage_pool_volume.go

```diff
@@ -184,4 +190,10 @@ type StorageVolumeSource struct {
 	//
 	// API extension: storage_api_volume_snapshots
 	VolumeOnly bool `json:"volume_only" yaml:"volume_only"`
+
+	// Project for the source volume
+	// Example: project1
+	//
+	// API extension: storage_api_project
+	Project string `json:"project,omitempty" yaml:"project,omitempty"`
```

Comment:
This change looks good but should go in a `shared/api: Add Project to StorageVolumeSource` commit instead.